### PR TITLE
Added Group Shape Fill options

### PR DESCRIPTION
**Pull request by drewfulton**
This has two components, first is accessing the fill properties from the GroupShape class and then second is to add a `.group()` option to the `FillFormat` object to allow the shape to be designated as having a group fill.

### CODE REVIEW

File: pptx/oxml/shapes/groupshape.py [hunk 1]
```diff
@@ -274,5 +280,24 @@ class CT_GroupShapeProperties(BaseOxmlElement):
         "a:extLst",
     )
     xfrm = ZeroOrOne("a:xfrm", successors=_tag_seq[1:])
+    eg_lineFillProperties = ZeroOrOneChoice(
```
Comment (luna215) [inline pptx/oxml/shapes/groupshape.py]: I'm curious why you named this lineFillProperties instead of grpFillProperties?

**Review by luna215 — COMMENTED** [review]
I left a simple question about naming. And I have a few general questions. How come you had to implement this option? Could we not iterate through the GroupShapes container and for each shape change the fill property? This implementation seems like we'll be able to change the fill for an entire group container to a single color/gradient/pattern. So if we have a Groupshape with a chart, textbox, and shape, and want to change GroupShape.fill.color to black, how would this look? I'm assuming that the color will be at the back of the GroupShape container so it doesn't interfere with the rest of the shapes.

File: pptx/oxml/shapes/groupshape.py (hunk 1 above)
Comment (drewfulton) [reply pptx/oxml/shapes/groupshape.py]: Apparently, I copied that block from the Line section and forgot to rename. Fixed!

**Comment by drewfulton** [comment]
So there are two aspects to the GroupShape Fill.  Basically, you can set a default fill property for the group and then for each Fill Object within the group, you have to set it's fill type to group.  That's why I added the `group()` function to the `FillFormat` group.  This group fill doesn't actually fill the group, it just sets a standard fill for everything in that group that is designated as a group fill.  

We haven't needed it, nor do I think we really do need it.  That's why I haven't added it to our helpers.  However, for the slide importer script, we need to have things as fully featured as possible so it can process these sorts of things from an incoming slide.  Before these additions, I could tell that individual items were set to GroupFill but I had no way to access the actual settings at the Group level.

**Review by luna215 — APPROVED** [review]
Ah, makes sense!

**Pull request merged**